src/Hydrators/IssueHydrator.php

```diff
@@ -2,14 +2,16 @@
 
 namespace ITBugTracking\Hydrators;
 
+use ITBugTracking\Entities\Comment;
+use ITBugTracking\Entities\IssueDetails;
 use PDO;
 use ITBugTracking\Entities\Issue;
 
 class IssueHydrator
 {
     public static function getIssues(PDO $db, $completedFilter): array|null
     {
-        $queryString = "SELECT `issues`.`id`,`issues`.`title`,LEFT(`issues`.`description`,100) AS 'summary',`issues`.`date_created`,COUNT(`comments`.`issue_id`) AS `comment_count`,`issues`.`completed`,`severities`.`name` AS `severity`
+        $queryString = "SELECT `issues`.`id`,`issues`.`title`,LEFT(`issues`.`description`,100) AS 'summary', DATE_FORMAT(`issues`.`date_created`, '%d/%m/%Y') AS 'date_created', COUNT(`comments`.`issue_id`) AS 'comment_count',`issues`.`completed`,`severities`.`name` AS 'severity'
```

Comment:
I would say this is actually a bad practice? 

Date formatting is one of the things that changes based on 1. where on earth you are 2. which language you speak 3. accessibility 4. user preference. When a story comes up that asks you to make dates localisable to users in different places, do you want to deal with changing your SQL queries?

It is generally a good practice to pull dates and times out of the database in the most complete and flexible format possible, so that you have the most options for users to configure how the dates are displayed later. Consider if the Issue and the IssueDetails had different date formatting. Keep it flexible until you have to send it to the user.

You have options:
* revert this change
* make a technical debt trello ticket and come back to this next story you deal with issues